frontend/subscriptions/index.js

```diff
@@ -0,0 +1,43 @@
+import { getCurrentRoute } from '@shopgate/pwa-common/helpers/router';
+import { login, disableLogin } from '@shopgate/engage/user';
+import {
+  appWillStart$, LoadingProvider, showModal, i18n,
+} from '@shopgate/engage/core';
+import {
+  requestSignInWithAppleAuthorization,
+  // requestSignInWithAppleCredentialState,
+} from '../classes';
+import { signInWithApple$ } from '../streams';
+import { fetchAppleConfig } from '../actions';
+
+/**
+ * @param {Function} subscribe The subscribe function.
+ */
+export default (subscribe) => {
+  subscribe(appWillStart$, async ({ dispatch }) => {
+    dispatch(fetchAppleConfig());
+    // const result = await requestSignInWithAppleCredentialState();
```

Comment:
Please remove commented out code.

Reply:
This will happen in the "log out" branch, since this is a preparation for it.